engine/time/src/main/java/io/deephaven/time/calendar/BusinessCalendarXMLParser.java

```diff
@@ -262,7 +270,8 @@ private static TimeRange<LocalTime>[] parseBusinessRanges(final List<Element> bu
 
             final LocalTime open = DateTimeUtils.parseLocalTime(openTxt);
             final LocalTime close = DateTimeUtils.parseLocalTime(closeTxt);
-            rst[i] = new TimeRange<>(open, close, true);
+            final boolean inclusiveEnd = Boolean.parseBoolean(includeCloseTxt); // defaults to false
```

Comment:
Should use `true` by default to match our previous behavior? (and fix the test cases by setting to `false` where appropriate...)

Reply:
`true` really doesn't match the prior behavior.  It yields a business time that is 1 ns longer.  `false` is closer to the prior behavior.

Reply:
The `TimeRange#nanos` being incorrect doesn't before doesn't change the fact that up until now, callers might be relying on parsing returning `io.deephaven.time.calendar.TimeRange#isInclusiveEnd` as `true`. 

Also, does this code path get hit when parsing a "legacy" calendar file?

Reply:
I've asked DHE to take a look at how this effects them.

Reply:
Good question on the legacy path.

These are the key parts for the legacy parsing of `TimeRange`:
* https://github.com/chipkent/deephaven-core/blob/timerange_off_by_one/engine/time/src/main/java/io/deephaven/time/calendar/BusinessCalendarXMLParser.java#L249
* https://github.com/chipkent/deephaven-core/blob/timerange_off_by_one/engine/time/src/main/java/io/deephaven/time/calendar/BusinessCalendarXMLParser.java#L294

The "legacy" and "new" `TimeRange` formats are parsed separately, so we could handle them differently if we want to.  Currently, legacy has `inclusiveEnd=true`.  This will result in a trading day that is 1ns longer than the current implementation, so in this case, I also advocate for setting `inclusiveEnd=false` so that the business day length does not change.  The legacy format for time ranges is not pure XML, so adding in inclusive controls is a little more messy, and I would prefer to avoid it.

I'm going to commit the  `inclusiveEnd=false` change for legacy, and I'm adding @rbasralian for thoughts on these changes since he deals with calendars.

Reply:
The PR description has been updated to reflect the breaking change.